evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -731,70 +744,46 @@ contract FluxAggregator is AggregatorInterface, Owned {
     return currentRound.add(1);
   }
 
-  function previousAndCurrentUnanswered(uint32 _id, uint32 _rrId)
+  function previousAndCurrentUnanswered(uint32 _roundId, uint32 _rrId)
     private
     view
     returns (bool)
   {
-    return _id.add(1) == _rrId && rounds[_rrId].updatedAt == 0;
+    return _roundId.add(1) == _rrId && rounds[_rrId].updatedAt == 0;
   }
 
-  function roundState(address _oracle)
-    external
+  function oracleRoundState()
```

Comment:
I would suggest enforcing that this is only callable by external accounts through
`require(tx.origin == msg.sender)`

Reply:
Could we do something like requiring that `msg.sender` be in the oracles mapping instead of relying on `tx.origin`?